packages/protocol/test/bridge/Bridge.test.ts

```diff
@@ -99,6 +114,12 @@ describe("Bridge", function () {
         ).deploy();
         await addressManager.init();
 
+        const signalService = await deploySignalService(
+            owner,
+            addressManager,
+            srcChainId
+        );
+
```

Comment:
```suggestion

```
Looks completely unused in every use of deployBridgeFixture()

Reply:
We still need it, but the returned `signalService` variable is indeed not used, but it doesn't hurt to make it available for the caller.